<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appPreferencesDialog</class>
 <widget class="QDialog" name="appPreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Умолчания</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основные настройки</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblOrganisation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;ЛПУ</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrganisation</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="CPolyclinicComboBox" name="cmbOrganisation"/>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="btnSelectOrganisation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblOrgStructure">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Подразделение</string>
         </property>
         <property name="buddy">
          <cstring>cmbOrgStructure</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRequestedNumbersQuantity">
         <property name="text">
          <string>&amp;Запрашиваемое количество</string>
         </property>
         <property name="buddy">
          <cstring>edtRequestedNumbersQuantity</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="edtRequestedNumbersQuantity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>61</width>
           <height>174</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCryptography">
      <attribute name="title">
       <string>&amp;Криптография</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbnOwnPK">
         <property name="text">
          <string>Ключ по &amp;СНИЛС</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="cmbCsp"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCsp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Крипто&amp;провайдер</string>
         </property>
         <property name="buddy">
          <cstring>cmbCsp</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbnCustomPK">
         <property name="text">
          <string>&amp;Произвольный ключ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="CCertComboBox" name="cmbUserCert"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFss">
      <attribute name="title">
       <string>Доступ к сервису &amp;СФР</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edtProxyAddress">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblServiceUrl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;URL сервиса</string>
         </property>
         <property name="buddy">
          <cstring>edtServiceUrl</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblFssCert">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Сертификат СФР</string>
         </property>
         <property name="buddy">
          <cstring>cmbFssCert</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edtProxyLogin">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="lblProxyPort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;порт</string>
         </property>
         <property name="buddy">
          <cstring>edtProxyPort</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="CCertComboBox" name="cmbFssCert">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="edtServiceUrl"/>
       </item>
       <item row="1" column="0" colspan="5">
        <widget class="QCheckBox" name="chkUseEncryption">
         <property name="text">
          <string>Использовать &amp;шифрование сообщений</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QCheckBox" name="chkUseProxy">
         <property name="text">
          <string>Использовать прокси-&amp;сервер</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblProxyAddress">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Адрес сервера</string>
         </property>
         <property name="buddy">
          <cstring>edtProxyAddress</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QSpinBox" name="edtProxyPort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>3128</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QCheckBox" name="chkProxyUseAuth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Прокси-сервер требует ау&amp;тентификацию</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblProxyLogin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Имя</string>
         </property>
         <property name="buddy">
          <cstring>edtProxyLogin</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="lblProxyPassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>па&amp;роль</string>
         </property>
         <property name="buddy">
          <cstring>edtProxyPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QLineEdit" name="edtProxyPassword">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CCertComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">library/CertComboBox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbOrganisation</tabstop>
  <tabstop>btnSelectOrganisation</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>edtRequestedNumbersQuantity</tabstop>
  <tabstop>cmbCsp</tabstop>
  <tabstop>rbnOwnPK</tabstop>
  <tabstop>rbnCustomPK</tabstop>
  <tabstop>cmbUserCert</tabstop>
  <tabstop>edtServiceUrl</tabstop>
  <tabstop>chkUseEncryption</tabstop>
  <tabstop>cmbFssCert</tabstop>
  <tabstop>chkUseProxy</tabstop>
  <tabstop>edtProxyAddress</tabstop>
  <tabstop>edtProxyPort</tabstop>
  <tabstop>chkProxyUseAuth</tabstop>
  <tabstop>edtProxyLogin</tabstop>
  <tabstop>edtProxyPassword</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseEncryption</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbFssCert</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseEncryption</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblFssCert</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
